### ISSUE TITLE
Update icon 

### ISSUE THREAD

**Issue by noamhoze** [Vayyar/vayyar-wp#1]
On vayyar.com, the icon is correct:
![image](https://user-images.githubusercontent.com/20975533/35193554-f8e80cfa-feac-11e7-81d2-1055e8f5f8b8.png)

On wp.vayyar.com it's wrong
![image](https://user-images.githubusercontent.com/20975533/35193576-4992dfcc-fead-11e7-9e68-b38a7ce478da.png)

**Comment by noamhoze** [comment]
@ygord-vayyar and @migkapa,
Please complete this task together.

**Comment by ygord-vayyar** [comment]
Note: this applies to both the home page and the About page.

**Comment by noamhoze** [comment]
@migkapa and @ygord-vayyar, 
I made a mistake, please restore it to the way it was.

This is the right one:
![35193576-4992dfcc-fead-11e7-9e68-b38a7ce478da](https://user-images.githubusercontent.com/20975533/35195754-be56592e-fed0-11e7-8029-9cc7ad9ff5f5.png)

**Comment by migkapa** [comment]
That section is used in home and about us. 
I added it as a global element. 
What I did is added an element called "Templetera" which makes possible editing and element just once and it's replicated to anywhere that element is added.

<img width="1097" alt="templetera" src="https://user-images.githubusercontent.com/2510443/35196466-ab856a40-fed2-11e7-98bc-fb57e7cbe4bd.png">
<img width="661" alt="templetera1" src="https://user-images.githubusercontent.com/2510443/35196469-abf85b86-fed2-11e7-8727-58f4a7722e79.png">
